### ISSUE TITLE
Logo seems informal

### ISSUE THREAD

**Issue by llrs** [cranchange/cran_change.org#17]
I think the logo of the "organization" could be a bit more professional. As it is is like a graffiti and does not inspire much confidence. I haven't found an official CRAN logo, but it could be like the CRAN letters with the R being the official R logo.

**Comment by llrs** [comment]
I have found, what I think is the official CRAN logo at https://cran.r-project.org/CRANlogo.png:
![CRANlogo](https://user-images.githubusercontent.com/6818218/119636868-5be11380-be15-11eb-8b84-e368165284d3.png)
Not sure about the license but maybe a modification and changing to green the Comprehensive word could be used for the form and the organization.

**Comment by MilesMcBain** [comment]
One thing I think the current logo has going for it is that it probably sets expectations appropriately. It was hastily put together, so was the letter, so was this GitHub org. There's very little going on behind the scenes here except for a handful of Slack messages. In that way it actually does represent where things are at right now.

Happy to consider alternative designs though!

**Comment by MilesMcBain** [comment]
Although I guess since `cran_histories` is also under the umbrella there is something a bit more deliberate going on here.

**Comment by wlandau** [comment]
I agree the logo reflects the current state of the draft, but I think a different design would be more conducive to the respectful dialogue that the letter proposes. By the time we reach out to CRAN, I expect the letter and the discussion around it to have gone through more phases of reframing and polishing. I think we all agree there is no immediate urgency and no deadline.

**Comment by strazto** [comment]
![image](https://user-images.githubusercontent.com/39424834/120138438-d8129700-c219-11eb-8f0d-1412149ba4f4.png)

Proposed alternative in case you wanted to go for less professional instead 😛 (only kidding of course)

**Comment by strazto** [comment]
awful jokes aside, in terms of the actual logo design as it is (putting it here so that this thread gives self contained context on what the logo is)

🔽 **Current logo** 
![current_logo](https://user-images.githubusercontent.com/39424834/120155384-1c129580-c234-11eb-9812-7a4f9bf5dfcb.png)



![changed logo](https://raw.githubusercontent.com/matthewstrasiotto/cran_change.org/logo/logo/cran_change.png)
🔼 **Replacement** *from the head of my forked branch [matthewstrasiotto/cran_change.org@logo](https://github.com/matthewstrasiotto/cran_change.org/logo)*

image description for current logo:

- Bottom layer: 
   the word "CRAN" in a bold, blocky san-serif font. 
   - The "R" is blue, similarly to the typical "R" logo. 
- Top layer:
   In a casual "grafitti" style typeface:
   - "H" is super imposed over the blue "R". 
   - "GE" is suffixed
   This alters the word to now spell "CHANGE" 


## Discussion of original design
   
To discuss the design- 
Very clever!

At present time, the "casual" approach of the "graffiti" font serves as a reflection of this being a community initiative, and is effective in this, as well as being fairly eye-catching.

To my mind, it (the typeface) also conjures the effect of:
- "rebellion". I'm unsure if that's the vibe the designer is intending or not
- Another potential intepretation is that of an antiestablishment, or dismantling of institutions
  - To me, that doesn't strongly align with the message of the letter - the goal is not to dismantle CRAN, but actual rather to improve communication, and clarify procedures
  
The voice my own judgement on the logo, I think that the overall idea is clever, and effective, however, I disagree with both the choice & colour of the typeface for the super-imposed text. 

## Suggestions

<details>
<summary>Suggestions that have now been incorporated</summary>

Though I've suggested that the "grafitti" style typeface might not convey the message, my main problem with it is just that i don't think it's easy to read, & I don't think the colour has enough contrast to make it very easily parsed.

I also think the GE are a little underemphasised and feel like an afterthought when they are an important part of the message.

My biggest actual suggestion would be 

- accessibility notwithstanding, up the saturation in "HGE" to a brighter red 
- Changing HGE to a more readable typeface, 
- Increasing HGE's emphasis
- Potentially scaling up the GE to slightly bigger than CRAN's text

</details>

I think for my updated logo, I'd maybe lower the weight of the **H** to ensure that the **R** is still legible

**Comment by strazto** [comment]
Guess I'll make a PR, but I've forked & put together something in inkscape:

![changed logo](https://raw.githubusercontent.com/matthewstrasiotto/cran_change.org/logo/logo/cran_change.png)

*from the head of my forked branch [matthewstrasiotto/cran_change.org@logo](https://github.com/matthewstrasiotto/cran_change.org/logo)*

**Comment by llrs** [comment]
I would love to know more about how CRAN and R developed to propose a more meaningful logo about the mission/goal it was set to solve. I know it was set up to check and distribute packages, but specifically why the R core took the burden to check the packages on multiple OS and versions would help.

Anyway, I think the logo should focus on the main attributes: "The" and "Comprehensive". CRAN changes a lot: new packages, policy updates and checks are frequently added or updated. But the focus of the letter is on making it easier to become **The** archive and to help make it more **Comprehnsive** (even though other archives exists like Bioconductor). If the logo could also reflect that it would be nice (sorry, my drawing skills are almost null and I don't have any concrete suggestion beyond what I already shared).

**Comment by strazto** [comment]
May I clarify -
Are you proposing changes to CRAN's logo, or the logo of cran_change project?

I would suggest that the best this project can do is something that evokes its mission - changing cran - This project doesn't aim to *replace* CRAN, and the logo for this project shouldn't be intended as the new logo for CRAN, but rather as something that suggests changes should be made to CRAN. 

In my opinion, there's pretty much nothing wrong with CRAN's logo - the acronym itself is recognizable to any intermediate R user, and I personally couldn't even remember what CRAN stood for (beyond "the R package repo") without looking it up today.

in summation:
The "what" of CRAN is: "Comprehensive R Archive Network" - Their logo can reflect that in whatever way, but the acronym CRAN is ubiquitous enough that any R user knows what it is. 

We are not designing a new logo for CRAN, but rather a logo for a project that seeks to build community support to influence CRAN's decisions.
The "what" of this project is: "Some things at CRAN could change"

If this project's logo was simply the logo that we want CRAN to adopt, any user who saw it would assume it was just another logo for CRAN, & not identify with a project that aims to influence CRAN's policy & operations.

**Comment by llrs** [comment]
@matthewstrasiotto When I opened this issue was to discuss the logo of this organization, not only cran_change.org and definitely not to change CRAN logo. 

What I'm proposing is to create the logo for the cranchange organization from CRAN's logo. Similar to what other organization such as the R ladies, R Foundation, R Hub... have done: Create their logo starting from another easily recognizable related to R.

**Comment by strazto** [comment]
Side note, does that svg i embedded looo okay? 
The font rendered well on my computer, but not so well on my phone it seems

**Comment by llrs** [comment]
I see this both on mobile and on computer:
![Screenshot of third comment from matthewstrasiotto https://github.com/cranchange/cran_change.org/issues/17#issuecomment-851320783 ](https://user-images.githubusercontent.com/6818218/120176631-ccb67000-c207-11eb-90ac-60d0c8d67dc6.png)

**Comment by wlandau** [comment]
I still think the red text and overlapping letters feel aggressive, a bit like we are invading their space. Is there a different way to approach this? Maybe a logo with blueprints, construction tape, a podium, or a megaphone?

**Comment by MilesMcBain** [comment]
@matthewstrasiotto I see the same as @llrs. Maybe render the font to a path in inkscape so it won't matter if we have it installed?

Also appreciate the dissection of my original design. :joy:

**Comment by strazto** [comment]
> I see this both on mobile and on computer:
> ![Screenshot of third comment from matthewstrasiotto https://github.com/cranchange/cran_change.org/issues/17#issuecomment-851320783 ](https://user-images.githubusercontent.com/6818218/120176631-ccb67000-c207-11eb-90ac-60d0c8d67dc6.png)
> 

Wow! Looks terrible

CHRANGE

**Comment by strazto** [comment]
> @matthewstrasiotto I see the same as @llrs. Maybe render the font to a path in inkscape so it won't matter if we have it installed?
> 
> Also appreciate the dissection of my original design. :joy: 

Yeah, I'll link to a png instead here, I expected that svg to be a little more portable

I probably wont decompose the font while I'm still editing it.
I promise it looks good on my end lmao (it's basically yours but the font is a little more conventional, and you can see the colour change already)

**Comment by strazto** [comment]
Exported & updated the pic!

**Comment by MilesMcBain** [comment]
It's an improvement IMHO!

**Comment by MilesMcBain** [comment]
@matthewstrasiotto I'd be happy to use your version for now if you can license it in a way that would allow that.

**Comment by strazto** [comment]
It's a fork of yours
Whatever license you want, or MIT

**Comment by strazto** [comment]
Need anything from me on the repo, or is me saying that enough?

**Comment by MilesMcBain** [comment]
We're not dealing with software here so I think that means we should probably use some kind of creative commons license. I can't see any issue with making the letter public domain, but maybe the logo should be cc-by?

**Comment by strazto** [comment]
okay, cc-by :) (i'm not particularly fussed)

**Comment by strazto** [comment]
or, maybe it should match whatever cran's logo's license is, since it's derivative work

**Comment by MilesMcBain** [comment]
@matthewstrasiotto is that because you used the typeface from the logo @llrs posted?

@llrs How did you even find that loo? I can't see it anywhere on the site.

**Comment by llrs** [comment]
I did a google search and then picked one from the official website. It is easier if you use `site:cran.r-project.org logo`.

**Comment by strazto** [comment]
@MilesMcBain i used some open font, but i tried as hard as possible to replicate the typeface from their site for the CRAN part. Gotta admit I'm not really clear on the specifics of it

**Comment by wlandau** [comment]
On reflection, do we need a logo at all? It carries risk and unknowns, and I am not sure exactly what value it adds to an effort like this one.

**Comment by MilesMcBain** [comment]
But then what will we put on the hex sticker?

**Comment by MilesMcBain** [comment]
I do take the point though :)

I am happy to put the logo on ice.

**Comment by MilesMcBain** [comment]
One thing though is that when I was reading the info for setting up a change.org petition, it recommends using an image because petitions that do that get X% more signatures. Probably not that big a deal, but it would be nice to have something to put there.

**Comment by wlandau** [comment]
Neat, I did not know that. Still, I am leaning toward omitting a logo. A hex logo is all in good fun for a package, but for a petition about a sensitive issue, I feel as though images are more easily misinterpreted, especially ones coming from non-graphic-designers like ourselves. We would open ourselves up to being viewed negatively in ways we cannot anticipate or control as easily as with words.
